fedimint-api/src/db/batch.rs

```diff
@@ -18,15 +18,15 @@ pub enum BatchItem {
     /// Insets new element, even if it already exists
     InsertElement(Element),
     /// Deletes element, errors if it doesn't exist
-    DeleteElement(Box<dyn DatabaseKeyPrefix + Send>),
+    DeleteElement(Box<dyn DbKeyToBytes + Send>),
```

Comment:
Could you please try to separate logic changes from renaming in the future (different commits in the same PR)? It's hard to focus on the relevant parts if it's mixed.

Reply:
That makes sense to me, I'll aim to do that next time.